src/main/java/com/laserfiche/repository/api/clients/impl/model/APIServerException.java

```diff
@@ -2,26 +2,175 @@
 @JsonInclude(JsonInclude.Include.NON_NULL)
 public class APIServerException {
 
     @JsonProperty("operationId")
-    public String operationId = null;
+    private String operationId = null;
 
     @JsonProperty("message")
-    public String message = null;
+    private String message = null;
 
     @JsonProperty("errorCode")
-    public Integer errorCode = null;
+    private Integer errorCode = null;
 
     @JsonProperty("errorClass")
-    public String errorClass = null;
+    private String errorClass = null;
 
     @JsonProperty("statusCode")
-    public Integer statusCode = null;
+    private Integer statusCode = null;
 
     @JsonProperty("errorSource")
-    public String errorSource = null;
+    private String errorSource = null;
+
+    public APIServerException operationId(String operationId) {
+        this.operationId = operationId;
+        return this;
+    }
+
+    @Schema(description = "")
+    public String getOperationId() {
+        return operationId;
+    }
+
+    public void setOperationId(String operationId) {
+        this.operationId = operationId;
+    }
+
+    public APIServerException message(String message) {
+        this.message = message;
+        return this;
+    }
+
+    @Schema(description = "")
+    public String getMessage() {
+        return message;
+    }
+
+    public void setMessage(String message) {
+        this.message = message;
+    }
+
+    public APIServerException errorCode(Integer errorCode) {
+        this.errorCode = errorCode;
+        return this;
+    }
+
+    @Schema(description = "")
+    public Integer getErrorCode() {
+        return errorCode;
+    }
+
+    public void setErrorCode(Integer errorCode) {
+        this.errorCode = errorCode;
+    }
+
+    public APIServerException errorClass(String errorClass) {
+        this.errorClass = errorClass;
+        return this;
+    }
+
+    @Schema(description = "")
+    public String getErrorClass() {
+        return errorClass;
+    }
+
+    public void setErrorClass(String errorClass) {
+        this.errorClass = errorClass;
+    }
+
+    public APIServerException statusCode(Integer statusCode) {
+        this.statusCode = statusCode;
+        return this;
+    }
+
+    @Schema(description = "")
+    public Integer getStatusCode() {
+        return statusCode;
+    }
+
+    public void setStatusCode(Integer statusCode) {
+        this.statusCode = statusCode;
+    }
+
+    public APIServerException errorSource(String errorSource) {
+        this.errorSource = errorSource;
+        return this;
+    }
+
+    @Schema(description = "")
+    public String getErrorSource() {
+        return errorSource;
+    }
+
+    public void setErrorSource(String errorSource) {
+        this.errorSource = errorSource;
+    }
+
+    @Override
+    public boolean equals(java.lang.Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
```

Comment:
I think it's better to use instanceof (it also eliminates the need for null check)

if (!(o instanecOf APIServerException)) {
}

Reply:
Hi Samad, all these code under /model are auto-generated by Swagger Codegen.

Reply:
Oh, sorry, I didn't notice that.